src/amo/components/LandingPage.js

```diff
@@ -108,9 +112,43 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = i18n.gettext(
+      oneLine`Change your browser's appearance. Choose from thousands of themes to give Firefox the
```

Comment:
It looks like we don't raise lint errors for lines that have strings in them over 100 columns? I don't get how that works but I guess that makes this easier.